app/src/main/java/com/kickstarter/ui/activities/MessagesActivity.java

```diff
@@ -76,15 +79,25 @@ protected void onCreate(final @Nullable Bundle savedInstanceState) {
       .compose(observeForUI())
       .subscribe(ViewUtils.setGone(this.backingInfoView));
 
-    this.viewModel.outputs.participantNameTextViewText()
+    this.viewModel.outputs.messageEditTextHint()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(this.participantNameTextView::setText);
+      .subscribe(this::setMessageEditTextHint);
 
     this.viewModel.outputs.messages()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(this.adapter::messages);
+      .subscribe(m -> {
+        this.adapter.messages(m);
+        this.recyclerView.invalidate();
```

Comment:
For posterity: we needed this line to redraw the recyclerview properly so that a newly sent message would render correctly, i.e. `alignParentEnd` of the view so it would show on the right